### DISCUSSION TITLE
Subscribe to the same feed multiple times

### DISCUSSION THREAD

**Discussion by VincenzoKoestler** [FreshRSS/FreshRSS#6656] [Q&A] [answered]
So I have been trying to add two web scraping feeds with the same base url because the newspaper has two different kind of articles I want to scrape. However, every time I want to add it it deletes all settings and states "feed already subscribed". Why can there only be one subscribtion to a feed? Any ideas for a workaround?

Newspage I am talking about is https://www.aerotelegraph.com. They have both regular Articles as well as "News Ticker"...

**Comment by Alkarex** [comment, marked as answer]
Interesting use-case, which I had not though about.
Quick untested response, but try `https://www.aerotelegraph.com/` and `https://www.aerotelegraph.com/#2`

**Reply by VincenzoKoestler** [reply]
That works! Thanks for the great and quick workaround idea :)
